lib/CL/CMakeLists.txt

```diff
@@ -279,6 +279,11 @@ if (ENABLE_LLVM)
     target_include_directories(lib_cl_llvm PRIVATE ${LLVM_SPIRV_INCLUDEDIR})
   endif()
 
+  if(ENABLE_HEADER_BUNDLING)
+    add_dependencies(lib_cl_llvm regenerate_bundled_headers)
```

Comment:
Beware that updates in `regenerate_bundled_headers` won't trigger recompilation of targets depending on `lib_cl_llvm`. `add_dependencies` just makes sure the `regenerate_bundled_headers` is built before `lib_cl_llvm` starts building.